frontend/tests/components/input-select.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { render } from '@testing-library/react';
+import { describe, expect, it, vi } from 'vitest';
+
+import { InputSelect } from '~/components/input-select';
+
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key: string) => key,
+  }),
+}));
```

Comment:
This can be removed because of <https://github.com/DTS-STN/future-sir/blob/main/frontend/tests/setup.ts#L5>.